tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -710,42 +722,108 @@ function setupStorybook(tree: Tree, options: NormalizedSchema) {
 
   return tree;
 }
+/**
+ * Remove function after migration is complete.
+ */
+function migrateE2ESetupToCypress(tree: Tree, options: NormalizedSchema) {
+  const e2ePath = joinPathFragments(options.projectConfig.root, 'e2e');
+  const e2eFolderExists = tree.exists(e2ePath);
 
-function moveStories(tree: Tree, options: NormalizedSchema) {
-  const componentName = names(options.normalizedPkgName).className.replace('React', '');
-  const sourceRoot = options.projectConfig.sourceRoot ?? '';
-  const oldStoriesPath = joinPathFragments(sourceRoot, 'stories');
-  const newStoriesPath = joinPathFragments(oldStoriesPath, componentName);
-  const storiesExistInNewPath = tree.exists(newStoriesPath);
+  if (!e2eFolderExists) {
+    return;
+  }
 
-  if (storiesExistInNewPath) {
+  visitNotIgnoredFiles(tree, e2ePath, treePath => {
+    if (treePath.includes('selectors.ts')) {
+      const newFilePath = joinPathFragments(options.paths.sourceRoot, 'testing', path.basename(treePath));
+
+      // Move testing helper file to src/testing.
+      tree.rename(treePath, newFilePath);
+    } else if (treePath.includes('.e2e.')) {
```

Comment:
no need for else if. in general if else should be considered as a codesmell. There is a lint rule which we should enable for our infra code. 

can we get rid of those in favour of exit early ? (`return`)

Reply:
[468a1fc](https://github.com/microsoft/fluentui/pull/24458/commits/468a1fc711f386f52572c0a507c927cedeac3d23)